### PR TITLE
web terminal image

### LINKED ISSUE #10030: Create dedicated WEB terminal image
Create a dedicated WEB terminal image with kubectl

### PR DESCRIPTION
**Pull request by zreigz**
**What does this PR do / Why do we need it**: new WEB terminal image with kubectl and helm applications

**Does this PR close any issues?**:
Fixes #10030

```release-note
NONE
```

### CODE REVIEW

File: hack/images/web-terminal/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.13
+LABEL maintainer="support@kubermatic.com"
+
+ENV KUBECTL_VERSION=v1.22.9 \
+    HELM_VERSION=v3.8.1
```
Comment (xrstf) [inline hack/images/web-terminal/Dockerfile]: If we start from scratch, can we start with the current Helm release (3.9.0)?

File: hack/images/web-terminal/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+RUN apk add --no-cache -U \
+    bash \
+    ca-certificates \
+    curl \
+    git \
+    iproute2 \
+    iptables \
```
Comment (xrstf) [inline hack/images/web-terminal/Dockerfile]: Do these two make sense if the user is not root?

File: hack/images/web-terminal/release.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+cd $(dirname $0)
+
+REPOSITORY=quay.io/kubermatic/web-terminal
```
Comment (xrstf) [inline hack/images/web-terminal/release.sh:21]: Please make a PR to https://github.com/kubermatic/access/blob/master/quay/kubermatic.yaml to create this quay repository.

Comment (zreigz) [reply hack/images/web-terminal/release.sh:21]: done https://github.com/kubermatic/access/pull/654

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**